### PR TITLE
Refactor/removing profile support

### PR DESCRIPTION
**Pull request by dgraeber**
### Description:

Remove support for profiles in orbit - replaced by podsettings

### Issue Reference URL

https://github.com/awslabs/aws-orbit-workbench/issues/782

----------
### Do not change content below. Mark applicable check boxes only.

Thank you for submitting a contribution to AWS Orbit Workbench.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a ISSUE associated with this PR?
- [ ] Does your PR title start with ISSUE-XXXX where XXXX is the issue number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [ ] Has your PR been rebased against the latest commit within the target branch (typically 'main')?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under Apache License 2.0?

### CODE REVIEW

File: cli/aws_orbit/models/context.py [hunk 1]
```diff
@@ -141,7 +141,7 @@ class TeamContext:
     jupyterhub_inbound_ranges: List[str]
     image: Optional[str]
     plugins: List[PluginManifest]
-    profiles: List[Dict[str, Union[str, Dict[str, Any]]]]
+    # profiles: Optional[List[Dict[str, Union[str, Dict[str, Any]]]]]
```
Comment (chamcca) [inline cli/aws_orbit/models/context.py:144]: shouldn't we just remove this?

File: cli/aws_orbit/models/context.py [hunk 2]
```diff
@@ -172,6 +172,7 @@ class TeamContext:
 
     # Manifest default
     authentication_groups: Optional[List[str]] = None
+    profiles: Optional[List[Dict[str, Union[str, Dict[str, Any]]]]] = None
```
Comment (chamcca) [inline cli/aws_orbit/models/context.py:175]: we leave this for backwards compatibility? did you try removing it? i think we have our SerDe configured to ignore extra parameters so it _should_ ignore any existing Profiles attributes

File: cli/aws_orbit/models/context.py [hunk 3]
```diff
@@ -406,11 +407,11 @@ def _load_context_from_manifest(manifest: "Manifest") -> Context:
             context.codeartifact_repository = manifest.codeartifact_repository
             context.cognito_external_provider = manifest.cognito_external_provider
             context.cognito_external_provider_label = manifest.cognito_external_provider_label
-            for team_manifest in manifest.teams:
-                team_context: Optional[TeamContext] = context.get_team_by_name(name=team_manifest.name)
-                if team_context:
-                    _logger.debug("Updating context profiles for team %s", team_manifest.name)
-                    team_context.profiles = team_manifest.profiles
+            # for team_manifest in manifest.teams:
```
Comment (chamcca) [inline cli/aws_orbit/models/context.py:410]: why commented instead of removed?

File: cli/aws_orbit/remote_files/cdk/team.py [hunk 4]
```diff
@@ -172,17 +172,19 @@ def __init__(
...
-            string_value="[]",
-            type=ssm.ParameterType.STRING,
-            description="Team additional profiles created by the team users",
-            parameter_name=ssm_profile_name,
-            simple_name=False,
-            tier=ssm.ParameterTier.INTELLIGENT_TIERING,
-        )
+
+
+#        ssm_profile_name = f"/orbit/{self.context.name}/teams/{self.team_name}/user/profiles"
```
Comment (chamcca) [inline cli/aws_orbit/remote_files/cdk/team.py:177]: commented not removed?

File: samples/manifests/demo/lake-admin-plugins.yaml [hunk 5]
```diff
@@ -25,9 +25,17 @@
         image: public.ecr.aws/v3o4w1g6/aws-orbit-workbench/utility-data:latest
         uid: 0
         gid: 0
--   PluginId: overprovisioning1
+-   PluginId: overprovisioning-primary-compute
```
Comment (chamcca) [inline samples/manifests/demo/lake-admin-plugins.yaml:28]: is the update to overprovisioning in this PR intentional? or did the branch just need updated?

File: cli/aws_orbit/models/context.py (hunk 1 above)
Comment (dgraeber) [reply cli/aws_orbit/models/context.py:144]: Yes...I left in for was of finding

File: cli/aws_orbit/models/context.py (hunk 2 above)
Comment (dgraeber) [reply cli/aws_orbit/models/context.py:175]: Needed for backward compatibility when I was flopping back-and-forth between versions...intended to remove it in 1.4 release

File: cli/aws_orbit/models/context.py (hunk 3 above)
Comment (dgraeber) [reply cli/aws_orbit/models/context.py:410]: placeholder...will remove

File: cli/aws_orbit/remote_files/cdk/team.py (hunk 4 above)
Comment (dgraeber) [reply cli/aws_orbit/remote_files/cdk/team.py:177]: will remove

File: samples/manifests/demo/lake-admin-plugins.yaml (hunk 5 above)
Comment (dgraeber) [reply samples/manifests/demo/lake-admin-plugins.yaml:28]: this is already in the main branch, so no impact

**Review by chamcca — APPROVED** [review]

**Pull request merged**